pkg/util/kubernetes/kubelet/kubelet_test.go

```diff
@@ -597,15 +597,12 @@ func (suite *KubeletTestSuite) TestKubeletInitTokenHttps() {
 	<-k.Requests // Throwing away first GET
 
 	assert.Equal(suite.T(), fmt.Sprintf("https://127.0.0.1:%d", kubeletPort), ku.kubeletClient.kubeletURL)
-	assert.Equal(suite.T(), "bearer fakeBearerToken", ku.kubeletClient.headers.Get("Authorization"))
```

Comment:
`transport.NewBearerAuthWithRefreshRoundTripper` handles the headers for us (asserted below)